wicker/core/storage.py

```diff
@@ -281,19 +332,159 @@ def get_column_concatenated_bytes_files_path(self, s3_prefix: bool = True, datas
             full_path = os.path.join(self.root_path, dataset_name, "__COLUMN_CONCATENATED_FILES__")
         else:
             full_path = os.path.join(self.root_path, "__COLUMN_CONCATENATED_FILES__")
-        if not s3_prefix:
-            return full_path.replace("s3://", "")
+        if prefix:
+            return full_path.replace(prefix, "")
         return full_path
 
-    def get_column_concatenated_bytes_s3path_from_uuid(self, file_uuid: bytes, dataset_name: str = None) -> str:
-        columns_root_path = self.get_column_concatenated_bytes_files_path(dataset_name=dataset_name)
-        return os.path.join(columns_root_path, str(uuid.UUID(bytes=file_uuid)))
-
     def get_temporary_row_files_path(self, dataset_id: DatasetID) -> str:
+        """Get path to temporary rows file path.
+
+        Args:
+            dataset_id (DatasetID): Id of dataset to use for pathing.
```

Comment:
Nit: consistent casing `Id` -> `ID`.

```suggestion
            dataset_id (DatasetID): ID of dataset to use for pathing.
```